Cannot find namespace 'ROSLIB'

I tried using the latest `roslib/index.d.ts` file in this repo and had a problem that I fixed; just want to let you know what the problem was and how I fixed it.

I got this error:

`error TS2503: Cannot find namespace 'ROSLIB'`

I'm using other external libraries, and @types for all the available definitions. I haven't had any problems with the type definition files for the other libraries. 

I fixed the issue by commenting out line 13 of the type definitions file, `export = ROSLIB`. I'm not sure if this was the best thing to do, but it fixed the error and everything seems to work ok.

--- a/roslib/index.d.ts
+++ b/roslib/index.d.ts
@@ -11,6 +11,7 @@
  ---------------------------------- */
 
 export = ROSLIB;
+export as namespace ROSLIB;
 
 declare namespace ROSLIB {
 	export class Ros {

